src/gcp_scanner/argparser.py

```diff
@@ -0,0 +1,82 @@
+import argparse
```

Comment:
Please run pylint in order to check for Python style violations.

Reply:
Have the updates ready in upcoming commit `49ca810`

Reply:
@mshudrak - sorry about this.. it's weird, I ran the pylint package locally on that file and it's telling me different spacing warnings.. taking the GitHub Actions build test comments into consideration, I hope the file is okay with the most recent commits...apologies if there are still tweaks to be made